jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -807,6 +807,12 @@ MethodHandle insertArguments(MethodHandle equivalent, MethodHandle unboxingHandl
 	 * equivalent for MethodHandle.
 	 */
 	private static final native MethodHandle getCPMethodHandleAt(Class<?> clazz, int index);
+	
+	/*
+	 * Get the class name from a constant pool class element, which is located at the
+	 * specified index in clazz's constant pool.
+	 */
+	private static final native String getClassNameAt(Class<?> clazz, int index);
```

Comment:
My apologies - I missed a naming convention issue.  All the other MH-constantPool related methods have a `CP` prefix - i.e.: `getCPClassNameAt`

see:
```
Java_java_lang_invoke_MethodHandle_getCPTypeAt
Java_java_lang_invoke_MethodHandle_getCPMethodTypeAt
Java_java_lang_invoke_MethodHandle_getCPMethodHandleAt
```

And Javadoc the method as well please.

Reply:
changed name to `getCPClassNameAt`, added Java doc for `getCPClassNameAt`, and resolved merge conflicts that arose from merging #19